test/test_utils.py

```diff
@@ -111,23 +111,43 @@ def test_a_models_dict_list_is_processed_correctly(self, get_models_list_fixture
 class TestOptionalColumn:
     """Test class to test the optional_column() functionality"""
 
-    def test_if_key_exists_value_is_returned_with_correct_width(self):
+    @pytest.mark.parametrize(
+        "value, result",
+        [
+            ("value", "value     "),
+            (1, "1         "),
+            (1.0, "1.0       "),
+            ({'sub_key': 'value'}, "{'sub_key': 'value'}"),
+            ({}, "{}        "),
```

Comment:
already confirms it passes, but probably worth adding in:
`([], "[]        "),` as if a dict is a default value, nothing saying an array couldn't be also

Reply:
Good idea. Added.